sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -188,7 +187,17 @@ export class TextAnalyticsClient {
     };
 
     const pipeline = createPipelineFromOptions(internalPipelineOptions, authPolicy);
-    this.client = new GeneratedClient(credential, this.endpointUrl, pipeline);
+    this.client = new GeneratedClient(
+      {
```

Comment:
I'd prefer to make a separate var declaration for the dummy token with a comment about why it's there.